tests/hook_main.py

```diff
@@ -1545,7 +1545,7 @@ def tax_delete(transaction):
         db.session.commit()
 
 
-@hooks.before("Tax > Get Taxes under an Event > Get Taxes under an Event")
+@hooks.before("Tax > Get Tax details under an Event > Get Tax details under an Event")
```

Comment:
What does this decorator do?

Reply:
It is used by dredd tests. It runs this method everytime it gets ```"Tax > Get Tax details under an Event > Get Tax details under an Event``` in the api blueprint file.
So basically we use this method to get ready for running a test.